api/src/main/resources/db/migration/v2020.09.18_15.00__issue_values_dict_metadata.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS pipeline.categorical_attributes(
```

Comment:
Could you please add issue number to sql file name?

Reply:
Done